package.json

```diff
@@ -58,6 +58,6 @@
     "typescript": "^4.0"
   },
   "publishConfig": {
-    "registry": "github:onfido/onfido-node"
```

Comment:
Using same setting as in the old version:
https://github.com/onfido/onfido-node/blob/a66154c972a28afabd19807084a0963951d5050d/package.json#L23C1-L24C1